plugins/geotiff/main.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+
+import rasterio
+from pyproj import Proj
+
+from vistas.core.gis.extent import Extent
+from vistas.core.plugins.data import RasterDataPlugin, VariableStats
+
+
+class GeoTIFF(RasterDataPlugin):
+
+    id = 'geotiff'
+    name = 'GeoTIFF Data Plugin'
+    description = 'Reads GeoTIFF (.tif) files'
+    author = 'Conservation Biology Institute'
+    version = '1.0'
+    extensions = [('tif', 'GeoTIFF')]
```

Comment:
Though less commonly used, GeoTIFFs may also have a `.tiff` extension.

http://reference.wolfram.com/language/ref/format/GeoTIFF.html.en
https://trac.osgeo.org/gdal/ticket/5124 (Apparently we're still living in the age of DOS 8)